src/script/user/UserRepository.ts

```diff
@@ -697,7 +698,6 @@ export class UserRepository {
     if (!user) {
       if (isMe) {
         userEntity.isMe = true;
-        this.userState.self(userEntity);
```

Comment:
No need to set the self user anymore, it's automatically found in the users list